### PR TITLE
check for platform_family instead of just platform

### PR DESCRIPTION
**Pull request by jsuchome**
It doesn't make that much sense here, but let's make it consistent with other parts...

### CODE REVIEW

File: chef/cookbooks/chef_handler/attributes/default.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 default["chef_handler"]["root_user"] = "root"
 
-case platform
+case platform_family
```
Comment (vuntz) [inline chef/cookbooks/chef_handler/attributes/default.rb:23]: Are you sure this is correct? I don't think `mac_os_x_server` is a platform?

Comment (jsuchome) [inline chef/cookbooks/chef_handler/attributes/default.rb:23]: I'm not sure. I think, the `mac_os_x_server` is a platform, but likely not platform family

**Comment by vuntz** [comment]
+1

**Comment by aplanas** [comment]
LGTM +1

**Pull request merged**